packages/fx-core/tests/plugins/resource/function/unit/scaffold.test.ts

```diff
@@ -43,6 +43,16 @@ const context: any = {
   root: path.join(__dirname, "ut"),
 };
 
+context.projectSettings = {
```

Comment:
Seems this setting isn't used by function ut. Could you help confirm and remove this if we don't need?